components/net/http_loader.rs

```diff
@@ -499,8 +480,7 @@ fn obtain_response(connector: &Pool<Connector>,
             debug!("Not notifying devtools (no request_id)");
             None
         };
-
-        return Ok((WrappedHttpResponse { response: response }, msg));
+        return Ok(({ response: HyperResponse }, msg));
```

Comment:
I was surprised that this builds, but I guess the type_ascription feature makes it valid. In any case, we should just use `Ok((response, msg))` instead.